source/object-model/classes-and-instances.md

```diff
@@ -59,6 +61,13 @@ var yehuda = Soldier.create({
 
 yehuda.say('Yes'); // alerts "Yehuda Katz says: Yes, sir!"
 ```
+In certain cases, you will want to pass arguments to `_super()` before or after overriding, so that the original method can continue operating as it normally would.  One common example is when overriding the `setupController()` hook in a route.  A handy shortcut for this is to use a "spread operator", like `...arguments`:
```

Comment:
Can you:

* break this onto multiple lines;
* add a line break before the code fences;
* change the example to something that doesn't involve a controller, since controllers are going away?

Thanks!